ts/Extensions/Annotations/Popup.ts

```diff
@@ -410,11 +444,22 @@ H.Popup.prototype = {
             }
         });
 
+        selectList.forEach(function (select: HTMLSelectElement): void {
+            const id = select.id;
+
+            // Get inputs only for the parameters, not for series and volume.
+            if (id !== PREFIX + 'select-series' && id !== PREFIX + 'select-volume') {
+                const parameter = select.id.split('highcharts-select-')[1];
```

Comment:
You can use the `id.split('...` (on variable) here instead of `select.id.split('...`